Specifications/QIR/Quantum-Runtime.md

```diff
@@ -38,6 +44,9 @@ The code that borrows the qubits guarantees that the state of the qubits when
 returned is identical, including entanglement, to their state when borrowed.
 It is always acceptable to satisfy `borrow` by allocating new qubits.
 
+Passing a null `%Qubit*` or `%Array*` to any of these functions should
```

Comment:
What about passing a pointer to a non-qubit array to `__quantum__rt__qubit_release_array`? (Per my other comment, maybe, reconsider providing this API at all?)